lib/vanagon/engine/hardware.rb

```diff
@@ -0,0 +1,102 @@
+require 'vanagon/engine/base'
+require 'json'
+require 'redis'
+
+REDIS_HOST = 'redis.delivery.puppetlabs.net'
```

Comment:
This makes this unusable for anyone else. This should be moved to some sort of configuration (.vanagonrc or something), or at the very least environment variables.